doc/lxc-attach.sgml.in

```diff
@@ -58,7 +58,6 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
       <arg choice="opt">-R, --remount-sys-proc</arg>
       <arg choice="opt">--keep-env</arg>
       <arg choice="opt">--clear-env</arg>
-      <arg choice="opt">-L, --pty-log <replaceable>file</replaceable></arg>
```

Comment:
This seems like a backward-incompatible change in behavior.  Any sites using lxc-attach -L in a script will end up with breakage?

Reply:
An argument has been added to `lxc_attach_options_t`.